Content.Shared/Doors/Systems/SharedDoorBoltSystem.cs

```diff
@@ -0,0 +1,54 @@
+using Content.Shared.Doors.Components;
+using Content.Shared.Popups;
+
+namespace Content.Shared.Doors.Systems;
+
+public abstract class SharedDoorBoltSystem : EntitySystem
```

Comment:
you need to have a an empty clientside system inheriting this for prediction to work